doc/source/whatsnew/v0.16.0.txt

```diff
@@ -169,6 +169,66 @@ Backwards incompatible API changes
      p // 0
 
 
+Indexing Changes
+~~~~~~~~~~~~~~~~
+
+.. _whatsnew_0160.api_breaking.indexing:
+
+The behavior of a small sub-set of edge cases for indexers have changed (:issue:`8613`):
+
+- to conform the intent of ``.loc`` to allow the same behavior for slice indexing as ``.ix`` when indexers are not found
```

Comment:
I would here also explain it in words, something like "slicing where the start and/or stop bound is not found in the index is now allowed (previously gave KeyError), as is the case for `ix`. The change is only for slicing, not when indecxing with a single label."